src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/proxymodule.cpp

```diff
@@ -168,7 +167,7 @@ REQUEST_NOTIFICATION_STATUS ASPNET_CORE_PROXY_MODULE::HandleNotificationStatus(R
 
 void ASPNET_CORE_PROXY_MODULE::SetupDisconnectHandler(IHttpContext * pHttpContext)
 {
-    auto connection = pHttpContext
+    auto* connection = pHttpContext
```

Comment:
Any particular reason you're adding the `*` to these?

Reply:
It's good from a readability perspective. `pHttpContext` is a pointer to reference and keeping `auto` just, one might be confused and will consider it as a reference! 
More info => https://clang.llvm.org/extra/clang-tidy/checks/readability-qualified-auto.html

Reply:
Are there other places where we use auto with a ptr? May want to make all instances follow the same format.

Reply:
I will get this checked in other places and c++ projects as well and will get fixed.
Btw I did not found any solution that will load native projects at once, I need to load vcproj one by one! 😑

Reply:
You should be able to open the entire sln by calling startvs.cmd in the src/Servers/IIS folder.

Reply:
Yeah, but that won't load C++ projects, it just C#!